logging/logging.go

```diff
@@ -180,7 +180,7 @@ func init() {
 	var err error
 	unixZeroTimestamp, err = ptypes.TimestampProto(time.Unix(0, 0))
 	if err != nil {
-		panic(err)
+		log.Fatalf("ptypes.TimestampProto(time.Unix(0,0)) failed: %v", err)
```

Comment:
For this panic I suggest just moving this logic to `NewClient` and return the error if it occurs

Reply:
Good pt. Afaik, `init` in Go is intended for variable initialization. And state checking should happen within the context of the function that actually depends on that state

Will change